### PR TITLE
pgbouncer

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**
this patch adds pgbouncer support to travis-hub, similarly to how it was added to api.

refs https://github.com/travis-pro/team-teal/issues/1787

### CODE REVIEW

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
Appears to be working on staging.

File: bin/sidekiq-pgbouncer [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+if [ ! -f "bin/start-pgbouncer-stunnel" ]; then
+  echo "warning: pgbouncer buildpack not found, setting PGBOUNCER_ENABLED=false"
+  export PGBOUNCER_ENABLED=false
+fi
+
+if [ "$PGBOUNCER_ENABLED" = '1' ] || [ "$PGBOUNCER_ENABLED" = 'true' ]; then
+  export PGBOUNCER_PREPARED_STATEMENTS=false
+  bin/start-pgbouncer-stunnel bin/sidekiq $@
+else
+  bin/sidekiq $@
```
Comment (meatballhat) [inline bin/sidekiq-pgbouncer]: I would expect these args to be passed via `"$@"` to ensure words aren't broken on whitespace.

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline bin/sidekiq-pgbouncer]: fixed 💜

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
Looks solid on prod as well. Merging!

**Pull request merged**